README.md

```diff
@@ -435,32 +435,56 @@ inductiva.tasks.list(5, status="success")
 
 ## Manage Resources
 
-**Inductiva API** provides a simple way to manage the hardware resources used to run the simulations. Users can launch virtual machines, list the available machines and terminate them. This is a feature available only to admins.
-In this way, users do not need to wait for their simulations in a queue and can have full control of the hardware used.
+**Inductiva API** provides a simple way to manage the hardware resources used to run simulations.
+This allows the user to have control over hardware that will be used exclusively for their experiments.
```

Comment:
By default, simulation requests are processed by a shared pool of machines serving multiple users. If you require dedicated resources, the Inductiva API allows you to easily set up virtual machines reserved solely for *your* simulations.